src/g_game.c

```diff
@@ -4502,151 +4502,150 @@ void doomprintf(player_t *player, msg_category_t category, const char *s, ...)
-  BIND_COMP(comp_doorstuck, 0, "Monsters get stuck on doortracks");
-  BIND_COMP(comp_pursuit,   1, "Monsters don't give up pursuit of targets");
-  BIND_COMP(comp_vile,      0, "Arch-Vile resurrects invincible ghosts");
-  BIND_COMP(comp_pain,      0, "Pain Elemental limited to 20 lost souls");
-  BIND_COMP(comp_skull,     0, "Lost souls get stuck behind walls");
+  BIND_COMP(comp_doorstuck, 0, "Monsters get stuck in door tracks");
+  BIND_COMP(comp_pursuit,   1, "Monsters can infight immediately when alerted");
+  BIND_COMP(comp_vile,      0, "Arch-Viles can create ghost monsters");
+  BIND_COMP(comp_pain,      0, "Pain Elementals are limited to 20 lost souls");
+  BIND_COMP(comp_skull,     0, "Lost souls can spawn past impassable lines");
   BIND_COMP(comp_blazing,   0, "Blazing doors make double closing sounds");
-  BIND_COMP(comp_doorlight, 0, "Tagged doors don't trigger special lighting");
+  BIND_COMP(comp_doorlight, 0, "Door lighting changes are abrupt");
   BIND_COMP(comp_god,       0, "God mode isn't absolute");
-  BIND_COMP(comp_skymap,    0, "Sky is unaffected by invulnerability");
+  BIND_COMP(comp_skymap,    0, "Don't apply invulnerability palette to skies");
   BIND_COMP(comp_floors,    0, "Use exactly Doom's floor motion behavior");
   BIND_COMP(comp_model,     0, "Use exactly Doom's linedef trigger model");
-  BIND_COMP(comp_zerotags,  0, "Linedef effects work with sector tag = 0");
-  BIND_COMP(comp_respawn,   0, "Creatures with no spawnpoint respawn at (0,0)");
-  BIND_COMP(comp_ledgeblock, 1, "Ledges block ground enemies");
-  BIND_COMP(comp_friendlyspawn, 1, "A_Spawn new thing inherits friendliness");
+  BIND_COMP(comp_zerotags,  0, "Linedef actions work on sectors with tag 0");
+  BIND_COMP(comp_respawn,   0, "Monsters not spawned at level start respawn at map origin");
+  BIND_COMP(comp_ledgeblock, 1, "Ledges block monsters");
+  BIND_COMP(comp_friendlyspawn, 1, "Things spawned by A_Spawn inherit friendliness of spawner");
   BIND_COMP(comp_voodooscroller, 0, "Voodoo dolls on slow scrollers move too slowly");
   BIND_COMP(comp_reservedlineflag, 1, "ML_RESERVED clears extended flags");
 
 #define BIND_EMU(id, v, help) \
   M_BindBool(#id, &overflow[(id)].enabled, NULL, (v), ss_none, wad_no, help)
 
-  BIND_EMU(emu_spechits, true, "SPECHITS overflow emulation");
-  BIND_EMU(emu_reject, true, "REJECT overflow emulation");
+  BIND_EMU(emu_spechits, true, "Emulate SPECHITS overflow");
+  BIND_EMU(emu_reject, true, "Emulate REJECT overflow");
   M_BindBool("emu_intercepts", &overflow[emu_intercepts].enabled, NULL, true,
-    ss_comp, wad_no, "INTERCEPTS overflow emulation");
-  BIND_EMU(emu_missedbackside, false, "Missed backside emulation");
-  BIND_EMU(emu_donut, true, "Donut overrun emulation");
+    ss_comp, wad_no, "Emulate INTERCEPTS overflow");
+  BIND_EMU(emu_missedbackside, false, "Emulate overflow caused by two-sided lines with missing backsides");
+  BIND_EMU(emu_donut, true, "Emulate donut overrun");
```

Comment:
"overrun" vs "overflow"

Reply:
Which one(s) are you talking about? The only one that IIRC didn't have a specific term in its description was the "missed backside" one; the other ones did, so I went with that.

Reply:
The donut case always had the word "overrun" in its description, the others have "overflow". Let be consistent and go with the latter.